src/brineconnector/client.py

```diff
@@ -1,17 +1,49 @@
 from .session import Session
-from .utils import params_to_dict
+from .utils import *
 from .bin.blockchain_utils import sign_msg
-from .exception import AuthenticationError
+from .exception import *
+import subprocess
 from typing import Optional, Union, List, Literal
-from .typings import Response, LoginResponse, FullDayPricePayload, CandleStickPayload, OrderBookPayload, RecentTradesPayload, ProfileInformationPayload, Balance, ProfitAndLossPayload, CreateOrderNoncePayload, CreateNewOrderBody, OrderPayload, CreateOrderNonceBody, CancelOrder, TradePayload, Order, TokenType
+from .typings import (
+    Response,
+    LoginResponse,
+    FullDayPricePayload,
+    CandleStickPayload,
+    OrderBookPayload,
+    RecentTradesPayload,
+    ProfileInformationPayload,
+    Balance,
+    ProfitAndLossPayload,
+    CreateOrderNoncePayload,
+    CreateNewOrderBody,
+    OrderPayload,
+    CreateOrderNonceBody,
+    CancelOrder,
+    TradePayload,
+    Order,
+    TokenType,
+    CoinStat,
+    CoinStatPayload
+)
+from web3 import Web3, Account
+from .constants import Config
+from decimal import Decimal
+from dotenv import load_dotenv
+import os
+load_dotenv()
 
+rpc_provider = os.environ['RPC_PROVIDER']
```

Comment:
Please don't add env variables in client (users wont be able to use this, please think from the perspective that youre installing a pip package), dotenv is only supposed to be show as an example. Check how its done in the nodejs connector, it should only be passed in as parameters.